app/controllers/comments_controller.rb

```diff
@@ -21,15 +24,19 @@ def edit
 
   # POST /comments or /comments.json
   def create
+  
```

Comment:
I am unable to comment under other people's photos, the reason might be because you didn't assign a photo to an instance variable and you didn't run authorize on the @photo  so that it can check if your user is able to comment. Here is how I did it:  

``` @photo = Photo.find(params[:comment][:photo_id])
    @comment = Comment.new(comment_params)
    authorize @photo, :show?
```